src/styling/Rocket.module.css

```diff
@@ -0,0 +1,3 @@
+.image {
+  width: 9rem;
+}
```

Comment:
- I would advice you to Kindly add some more stying, and follow the framework provided.

![Screenshot from 2023-03-14 20-44-25](https://user-images.githubusercontent.com/55842414/225097352-a431c6e0-52df-4eb6-94e3-cf51870137b4.png)